Common/src/main/java/com/blamejared/crafttweaker/api/item/ExpandIIngredientWithAmount.java

```diff
@@ -0,0 +1,27 @@
+package com.blamejared.crafttweaker.api.item;
+
+import com.blamejared.crafttweaker.api.annotation.ZenRegister;
+import com.blamejared.crafttweaker.api.data.IData;
+import com.blamejared.crafttweaker.api.data.IntData;
+import com.blamejared.crafttweaker.api.ingredient.IIngredient;
+import com.blamejared.crafttweaker.api.ingredient.IIngredientWithAmount;
+import com.blamejared.crafttweaker.api.tag.expand.ExpandItemTag;
+import com.blamejared.crafttweaker.api.tag.type.KnownTag;
+import com.blamejared.crafttweaker.api.util.Many;
+import com.blamejared.crafttweaker_annotations.annotations.Document;
+import net.minecraft.world.item.Item;
+import org.openzen.zencode.java.ZenCodeType;
+
+@ZenRegister
+@Document("vanilla/api/item/ExpandIIngredientWithAmount")
+@ZenCodeType.Expansion("crafttweaker.api.ingredient.IIngredientWithAmount")
+public class ExpandIIngredientWithAmount {
```

Comment:
Why expand this compared to just having the method in the class as a default method?

Reply:
Fixed.